src/Client/Controller/Controller.cpp

```diff
@@ -67,46 +84,55 @@ void Controller::OnLogin(LoginInDTO in_dto)
 	m_http_client.set_auth_scheme(Net::AUTH_SCHEME_TYPE_BEARER);
 	m_http_client.set_token(out_dto.token);
 
-	// there i could set new data to the page
-
+	SetMainSubPage(MainSubPages::LISTS);
 	m_main_window.SetCurrentPage(UIPages::MAIN);
 }
 
-void Controller::OnGetLists()
+void Controller::OnLogout()
 {
-	ListsModel model;
-	ListInDTO in_dto;
-	auto request  = model.FormRequest(in_dto);
-	auto response = m_http_client.Request(request);
+	m_http_client.ReleaseToken();
+	m_main_window.SetCurrentPage(UIPages::LOGIN);
+}
 
-	// checking response status
-	// check if status ok
-	// errorstatus
-	if(response.status >= Poco::Net::HTTPResponse::HTTP_BAD_REQUEST)
-	{
-		if(response.status == Poco::Net::HTTPResponse::HTTP_UNAUTHORIZED)
-		{
-			m_main_window.m_login_page.ChangeLoginErrorLabel(
-					"Login and password do not match");
-		}
-		else
-		{
-			QMessageBox::information(&m_main_window
-					, QString::fromStdString("Login failed!")
-					, QString::fromStdString(response.reason));
-		}
-		return;
+void Controller::SetMainSubPage(MainSubPages page)
+{
+	switch(page) {
+	case MainSubPages::LISTS:
+		UpdateMainListsSubPage();
+		break;
+	case MainSubPages::CREATE_LIST:
+		break;
+	case MainSubPages::PRODUCTS:
+		break;
+	case MainSubPages::ICOMES:
```

Comment:
i believe this is a typo
```suggestion
	case MainSubPages::INCOMES:
```